tests/puppeteer/campaigns/functional/BO/13_shopParameters/03_productSettings/02_productPage/05_separatorOfAttributeAnchor.js

```diff
@@ -0,0 +1,88 @@
+require('module-alias/register');
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_productSettings_separatorOfAttributeAnchorOnProductLinks';
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const ProductSettingsPage = require('@pages/BO/shopParameters/productSettings');
+const ProductPage = require('@pages/FO/product');
+const HomePage = require('@pages/FO/home');
+const SearchResultsPage = require('@pages/FO/searchResults');
+// Importing data
+const {Products} = require('@data/demo/products');
+
+let browser;
+let page;
+const attributesWithSeparator2 = '1,size,s/8,color,white';
+const attributesWithSeparator1 = '1-size-s/8-color-white';
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    productSettingsPage: new ProductSettingsPage(page),
+    homePage: new HomePage(page),
+    productPage: new ProductPage(page),
+    searchResultsPage: new SearchResultsPage(page),
+  };
+};
+
+describe('Test separator of attribute anchor on the product links', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+
+  // Login into BO and go to product settings page
+  loginCommon.loginBO();
+
+  it('should go to \'Shop parameters > Product Settings\' page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToProductSettingsPage', baseContext);
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.shopParametersParentLink,
+      this.pageObjects.boBasePage.productSettingsLink,
+    );
+    const pageTitle = await this.pageObjects.productSettingsPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.productSettingsPage.pageTitle);
+  });
+
+  const tests = [
+    {args: {option: ',', attributesInProductLink: attributesWithSeparator2}},
+    {args: {option: '-', attributesInProductLink: attributesWithSeparator1}},
```

Comment:
You can use what I described earlier here:
```suggestion
    {args: {option: ',', attributesInProductLink: attributesWithSeparator.join(',')}},
    {args: {option: '-', attributesInProductLink: attributesWithSeparator.join('-')}},
```
You could even make it automatic with a second array providing the separators.